school/mainpage/models.py

```diff
@@ -1,3 +1,9 @@
-# from django.db import models
+from django.db import models
+
 
 # Create your models here.
+class Logger(models.Model):
+    method = models.CharField(max_length=10)
+    path = models.URLField(max_length=200)
```

Comment:
https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers